devices/opi-pc-plus/kali-rolling-armhf/pieman.yml

```diff
@@ -0,0 +1,14 @@
+# Maintainer: Polina Popova <ppopova.wis@gmail.com>
+
+kali-rolling-armhf:
```

Comment:
pieman.yml is missing the following parts:
* `includes: systemd`
  systemd is not a part of the base system, so you have to specify it explicitly. Otherwise, the target system won't have the possibility to boot.
* ```  
  params:
    - ENABLE_GOOGLE_DNS
  ```
  You have to configure DNS. Otherwise, the target system will be cut off from the Internet.